new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleTraefikValidation.java

```diff
@@ -0,0 +1,685 @@
+    }
+
+    // uninstall ingress
+    if (ingressParam != null) {
+      assertThat(uninstallIngress(ingressParam))
+          .as("Test uninstallIngress returns true")
+          .withFailMessage("uninstallIngress() did not return true")
+          .isTrue();
+    }
+
+    // uninstall traefik release
+    if (tfHelmParams != null) {
+      assertThat(uninstallTraefik(tfHelmParams))
+          .as("Test uninstallTraefik returns true")
+          .withFailMessage("uninstallTraefik() did not return true")
+          .isTrue();
+    }
+
+    // Delete traefik Namespace
+    if (tfNamespace != null) {
+      assertThatCode(
+          () -> deleteNamespace(tfNamespace))
+          .as("Test that deleteNamespace does not throw an exception")
+          .withFailMessage("deleteNamespace() threw an exception")
+          .doesNotThrowAnyException();
+      logger.info("Deleted namespace: {0}", tfNamespace);
+    }
+  }
+
+  /**
+   * prepare and install weblogic operator
+   */
+  private static void installingOperator() {
+    // get a new unique opNamespace
+    logger.info("Creating unique namespace for Operator");
+    opNamespace = assertDoesNotThrow(() -> createUniqueNamespace(),
+        "Failed to create unique namespace due to ApiException");
+    logger.info("Created a new namespace called {0}", opNamespace);
+
+    logger.info("Creating unique namespace for Domain");
+    domainNamespace = assertDoesNotThrow(() -> createUniqueNamespace(),
+        "Failed to create unique namespace due to ApiException");
+    logger.info("Created a new namespace called {0}", domainNamespace);
+
+    // Create a service account for the opNamespace
+    logger.info("Creating service account");
+    String serviceAccountName = opNamespace + "-sa";
+    assertDoesNotThrow(() -> createServiceAccount(new V1ServiceAccount()
+        .metadata(
+            new V1ObjectMeta()
+                .namespace(opNamespace)
+                .name(serviceAccountName))));
+    logger.info("Created service account: {0}", serviceAccountName);
+
+    // helm install parameters
+    opHelmParams = new HelmParams()
+        .releaseName(OPERATOR_RELEASE_NAME)
+        .namespace(opNamespace)
+        .chartDir(OPERATOR_CHART_DIR);
+
+    // Operator chart values to override
+    OperatorParams opParams =
+        new OperatorParams()
+            .helmParams(opHelmParams)
+            .image(OPERATOR_IMAGE)
+            .domainNamespaces(Arrays.asList(domainNamespace))
+            .serviceAccount(serviceAccountName);
+
+    // install Operator
+    logger.info("Installing Operator in namespace {0}", opNamespace);
+    assertTrue(installOperator(opParams),
+        String.format("Operator install failed in namespace %s", opNamespace));
+    logger.info("Operator installed in namespace {0}", opNamespace);
+
+    // list helm releases
+    logger.info("List helm releases in namespace {0}", opNamespace);
+    helmList(opHelmParams);
```

Comment:
you don't do anything with the results? is this jut for logging purposes?

Reply:
added the verification method